packages/interfaces/src/waku.ts

```diff
@@ -251,6 +270,14 @@ export interface IWaku {
    */
   createEncoder(params: CreateEncoderParams): IEncoder;
 
+  /**
+   * Sends a message to the Waku network.
+   *
+   * @param {ISendMessage} message - The message to send.
+   * @returns {Promise<RequestId>} A promise that resolves to the request ID
+   */
+  send(message: ISendMessage): Promise<RequestId>;
```

Comment:
the Waku message Id must be exposed in the API to enable SDS's retrieval hints.

I strongly recommend for it to be returned by `send` to avoid issues with timestamp and pubsubtopic (both being used to calculate message id)